chronosphere/tfschema/derived_label.go

```diff
@@ -127,6 +127,27 @@ var DerivedLabel = map[string]*schema.Schema{
 			},
 		},
 	},
+	"span_tag": {
+		Type:     schema.TypeList,
+		MaxItems: 1,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"name_mappings": {
```

Comment:
Note: this should be `name_mapping` (singular) per TF naming convention, however there is already a `name_mappings` field under `metric_label`, so opting to stay consistent within the entity.